src/library/mixin_helper.lua

```diff
@@ -561,6 +562,24 @@ function mixin_helper.to_fcstring(value, fcstr)
     return fcstr
```

Comment:
In keeping with the comment below on `to_string`, the line above this one (I can't target it with a review comment) should read:
```lua
fcstr.LuaString = value == nil and "" or tostring(value)
```